src/partner-access/partner-access.service.ts

```diff
@@ -97,20 +100,19 @@ export class PartnerAccessService {
   ): Promise<PartnerAccessEntity> {
     const partnerAccess = await this.getValidPartnerAccessCode(partnerAccessCode);
 
-    partnerAccesses.map(async (pa) => {
-      if (partnerAccess.partner.id === pa.partner.id && pa.active === true) {
```

Comment:
Could we have a chat about this? I want to make sure I'm clear on the use case for multiple codes for the same partner. 

The use case I have in mind is - we add a Bumble user with no feature for  therapy but they do have live chat - they get a partner access code we generate for them. Then later if Bumble want to add therapy, the user will be given a new access code with the therapy feature.

Reply:
So the two codes together show what the Bumble user can do in Bloom